README.md

```diff
@@ -467,6 +469,8 @@ IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2021. [You
 - <a name="Sengupta21ciss0"></a>Sengupta, J. P., Villemur, M., Andreou, A. G.,  
 *[Efficient, event-driven feature extraction and unsupervised object tracking for embedded applications](https://doi.org/10.1109/CISS50987.2021.9400234)*,  
 55th Annual Conf. on Information Sciences and Systems (CISS), 2021.
+- [Alzugaray, I., Ph.D. Thesis, 2022](#Alzugaray22PhD),  
```

Comment:
For some reason this link does not show up in the rich difference view of the pull request. I will check right after merging.